### PR TITLE
use sidekiq-cron instead of whenever

### PR DESCRIPTION
**Pull request by macool**

### CODE REVIEW

**Comment by macool** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;gYPar1oC&#x2F;650-qae-support-server-maintenance"> QAE Support: Server Maintenance</a>

File: app/workers/scheduled/performance_platform_service_worker.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+module Scheduled
+  class PerformancePlatformServiceWorker < BaseWorker
+    def perform
+      ::CronJob.run("performance") do
```
Comment (matthewford) [inline app/workers/scheduled/performance_platform_service_worker.rb]: Can remove this class - it was used for locking - it is no longer needed

File: config/initializers/sidekiq-cron.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+production_schedule = {
+  "performance_platform_service" => {
+    "cron" => "0 0 * * 0",
+    "class" => "Scheduled::PerformancePlatformServiceWorker"
+  }
+}
+
+Sidekiq::Cron::Job.load_from_hash(default_schedule)
+if Rails.env.production?
```
Comment (matthewford) [inline config/initializers/sidekiq-cron.rb]: Probably want to use an ENV Variable as all apps will run in production env

File: Gemfile.lock [hunk 3]
```diff
@@ -413,6 +412,8 @@ GEM
     redis-activesupport (4.0.0)
       activesupport (~> 4)
       redis-store (~> 1.1.0)
+    redis-namespace (1.5.2)
```
Comment (matthewford) [inline Gemfile.lock]: :( do we need this?

Comment (macool) [inline Gemfile.lock]: @matthewford I upgraded sidekiq but it's a dependency for `sidekiq-cron` gem

**Pull request merged**